pages/docs/tutorials/message-validation-for-kafka.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Message validation for Kafka
+description: The goal of this tutorial is to help you validate messages for Kafka.
+weight: 100
+---
+
+# Message Validation for Kafka 
+
+## Introduction
+
+[comment:] <> (What do we learn, and at the end, what do we accomplish? )  
+
+Validating messages with Kafka involves defining the message schemas using AsyncAPI, and then using those schemas to validate incoming and outgoing messages in your Kafka-based applications. 
+
+In this tutorial, you'll learn how to validate messages for Kafka. In addition, you'll learn more about EDAs, and the role of Kafka in validating messages.
+
+By the end of this tutorial, you will:
+* 
+* 
+* 
+
+## Background context
+	Define a message.
+	What is Kafka?
+	Why do people use Kafka with AsyncAPI?
+
+### Prerequisites
+
+[comment:] <> (What does the reader need to accomplish this task?)
+
+
+## Installation / Validation
+	
+### Validating at the Broker level
+
+### Validating at the client level
+
+
+### Summary
+
+[comment:] <> (Here is what we learned and did.)
```

Comment:
`Summary`
- What did we learn, what we built, and how to apply this to real life.